tools/inference_gradio.py

```diff
@@ -46,11 +45,18 @@ def infer(
 def get_args() -> Namespace:
     """Get command line arguments.
 
+    Example:
+
+        >>> python tools/inference_gradio.py --model ./anomalib/models/padim/config.yaml --weight_path ./results/padim/mvtec/bottle/weights/model.ckpt
+        >>> python tools/inference_gradio.py --model padim --weight_path ./results/padim/mvtec/bottle/weights/model.ckpt
+
     Returns:
         Namespace: List of arguments.
     """
     parser = ArgumentParser()
-    parser.add_argument("--config", type=Path, required=True, help="Path to a model config file")
```

Comment:
Maybe we could keep `config` to be consistent with other entrypoints?

Reply:
I wasn't sure what to call it. We can either pass the model config or the model name now. If I switch back to config should I drop calling inferencer with just model name?

Reply:
`--config` would be inline with the new PL CLI, so I would prefer that.

> If I switch back to config should I drop calling inferencer with just model name?

Not sure if I get this part

Reply:
What I mean is that initially with `config` parameter we had to pass the `yaml` file to the inferencer. I changed it so that we can either pass `yaml` or only the model name. So, I wasn't sure sure what to call this parameter. If I change it back to `config` then passing just the model name might not match with the parameter name. In which case we can drop passing only the model name and keep `yaml` as the only option. It might be better in some sense as it will force people to ensure that their train config matches with the config they use for inference. Otherwise inferencer might pick up the default config with just the model name which might not match with the path for config that was used to train with.

Reply:
I agree. Passing only config would ensure the right config file is passed. Otherwise, it would be the default config, which may not be the same as the one that is used to train the model.